frontend/src/static/css/style.scss

```diff
@@ -244,6 +244,10 @@ header{
             color:mui-color("grey");
             text-decoration:none;
         }
+
+        &--contribution{
+            font-size:0.65rem;
```

Comment:
since they use the same font-size, would prefer to use a separate class that applies the font size to all relevant elements.